db/migrate/201904101430_fix_mtu_min_value.rb

```diff
@@ -0,0 +1,8 @@
+class FixMTUMinimumValue < ActiveRecord::Migration[5.2]
```

Comment:
Looks like tests are failing because they expect the class name to match the file name, so this should be 
```suggestion
class FixMtuMinValue < ActiveRecord::Migration[5.2]
```